planemo/commands/cmd_project_init.py

```diff
@@ -43,11 +43,11 @@ def cli(ctx, path, template=None, **kwds):
         untar_args = UNTAR_ARGS % (tempdir)
         untar_to(DOWNLOAD_URL, tempdir, untar_args)
         template_dir = os.path.join(tempdir, template)
-        shell("ls '%s'" % (template_dir))
-        shell("mv '%s'/* '%s'" % (template_dir, path))
+        shell(['ls', template_dir])
+        shell(['mv', "%s/*" % template_dir, path])
```

Comment:
Thanks for the fix - the rest of this looks good but this line required the shell expansion to happen. Can I revert just this line or are you working on an approach that avoids shell to do this?

Reply:
Yep, this broke project_init tests. Rebased with a fixed (and shorter) version.